README.md

```diff
@@ -89,6 +94,15 @@ Add tags to your constructs. We recommend setting an `env` and `service` tag to
 
 The Datadog CDK construct takes in a list of lambda functions and installs the Datadog Lambda Library by attaching the Lambda Layers for [Node.js][2] and [Python][1] to your functions. It redirects to a replacement handler that initializes the Lambda Library without any required code changes. Additional configurations added to the Datadog CDK construct will also translate into their respective environment variables under each lambda function (if applicable / required).
 
+## Resources to learn about CDK
+
+CDK is to CloudFormation Macro what programming languages is to assembly. Constructs are building blocks of CDK and Construct libraries represent arrangement of AWS resources. 
```

Comment:
I think it's a great idea to include some info here for people stumbling on this repo! However, I would suggest a couple changes:
- I don't think CDK has a relationship to CloudFormation macros per se -- I think the relationship is more to a CloudFormation template, not a macro. A CDK project compiles into a CloudFormation template much like code written in a programming language can compile into assembly code. But CloudFormation macros are something else -- they are essentially Lambda functions that can transform a CloudFormation template. They don't really relate to CDK as far as I know.
- Programming languages `is` to assembly -> Programming languages `are` to assembly.
- "Construct libraries represent arrangement of AWS resources" sounds a little odd to me, maybe we can reword that to "Construct libraries make it easy to add encapsulated components consisting of multiple AWS resources to your application".

I realize you pulled this from our internal wiki page, so this should probably be updated there as well. Particularly the bit about the macro.